src/app/pages/utils/isPortraitVideo/fixtures.ts

```diff
@@ -0,0 +1,205 @@
+import { OptimoBlock } from '#app/models/types/optimo';
+
+export const mediaBlockWithPV: OptimoBlock[] = [
+  {
+    id: '326c14fc',
+    type: 'aresMedia',
+    model: {
+      blocks: [
+        {
+          id: 'b6ceabfb',
+          blockId: 'urn:bbc:ares::clip:p0l0sm0w',
+          type: 'aresMediaMetadata',
+          model: {
+            id: 'p0l0sm0w',
+            subType: 'clip',
+            format: 'video',
+            title: 'test',
+            synopses: {
+              short: 'test',
+              long: 'test',
+              medium: 'test',
+            },
+            imageUrl: 'test',
+            embedding: true,
+            advertising: true,
+            versions: [
+              {
+                versionId: 'p0l0sm10',
+                types: ['Portrait'],
+                duration: 88,
+                durationISO8601: 'PT1M28S',
+                warnings: {},
+                availableTerritories: {
+                  uk: true,
+                  nonUk: true,
+                },
+                availableFrom: 1743098855000,
+              },
+            ],
+            syndication: {
+              destinations: ['default'],
+            },
+            smpKind: 'programme',
+            webcastVersions: [],
+          },
+          position: [2, 2, 1],
+        },
+      ],
+    },
+    position: [2, 2],
+  },
+];
+
+export const mediaBlockWithMultipleVersionsTypes: OptimoBlock[] = [
+  {
+    id: '326c14fc',
+    type: 'aresMedia',
+    model: {
+      blocks: [
+        {
+          id: 'b6ceabfb',
+          blockId: 'urn:bbc:ares::clip:p0l0sm0w',
+          type: 'aresMediaMetadata',
+          model: {
+            id: 'p0l0sm0w',
+            subType: 'clip',
+            format: 'video',
+            title: 'test',
+            synopses: {
+              short: 'test',
+              long: 'test',
+              medium: 'test',
+            },
+            imageUrl: 'test',
+            embedding: true,
+            advertising: true,
+            versions: [
+              {
+                versionId: 'p0l0sm10',
+                types: ['Editorial', 'Some Invalid Type', 'Portrait'],
```

Comment:
We can remove this - let's not add it to fixture data as it is not realistic. If you want to add an invalid type like this, perhaps it's best to override it in the test itself. 

```suggestion
                types: ['Editorial', 'Portrait'],
```